ooyala_player/ooyala_player.py

```diff
@@ -262,112 +216,40 @@ class OoyalaPlayerBlock(OoyalaPlayerMixin, XBlock):
 
     content_id = String(
         display_name=_("Content Id"),
-        help=_("Identifier for the Content Id."),
+        help=_("Brightcove Video ID."),
         scope=Scope.content,
         default='6110589153001'
     )
 
-
     reference_id = String(
         display_name=_("Reference Id"),
         help=_("Reference ID for the Content Id."),
         scope=Scope.content,
         default=''
     )
 
-    transcript_file_id = String(
```

Comment:
@naeem91 removing these fields will impact existing blocks?

Reply:
It won't impact existing blocks. The saved data in removed fields will just stay there in DB and newly created blocks will only have the necessary new fields.